selfdrive/car/hyundai/values.py

```diff
@@ -126,6 +127,7 @@ class HyundaiCarInfo(CarInfo):
   ],
   CAR.VELOSTER: HyundaiCarInfo("Hyundai Veloster 2019-20", "Smart Cruise Control (SCC)", min_enable_speed=5. * CV.MPH_TO_MS, harness=Harness.hyundai_e),
   CAR.SONATA_HYBRID: HyundaiCarInfo("Hyundai Sonata Hybrid 2020-22", "All", harness=Harness.hyundai_a),
+  CAR.IONIQ_5: HyundaiCarInfo("Hyundai Ioniq 5 2022", "HDA2", harness=Harness.none),
```

Comment:
```suggestion
  CAR.IONIQ_5: HyundaiCarInfo("Hyundai Ioniq 5 2022", "Highway Driving Assist 2", harness=Harness.none),
```

Let's use the full market name, as this is now shown on the website.